Extensions/retags.js

```diff
@@ -53,7 +53,7 @@ XKit.extensions.retags = {
 	tag_popover: function(elements) {
 		$(elements).each(function() {
 			var $element = $(this), retagClass, $target, host, id;
-			if ($element.find('div.retags').length) {
+			if ($element.find('div.retags').length || $element.find('p.note-added-tags').length) {
```

Comment:
I *think* the more performant thing here is actually `element.find('div.retags, p.note-added-tags')`, since the browser can optimize the selector.